cnc_ctrl_v1/Motion.cpp

```diff
@@ -67,6 +76,8 @@ void movementUpdate(){
   movementUpdated = true;
 }
 
+
+// why does this return anything
```

Comment:
I think that there was a plan to use the return value to check if the move had completed or if it was unable to reach it's target...but that is long gone so it could be a void() at this point